en/02_Developer_Guides/07_Debugging/01_Error_Handling.md

```diff
@@ -164,6 +164,9 @@ The calls key, `MailHandler`, can be anything you like: its main purpose is to l
 
 ### Logging to a file
 
+> [!WARNING]
+> You will need to make sure the user running the PHP process has write access to the log file, wherever you choose to put it.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.